rendering.js

```diff
@@ -119,10 +121,16 @@ function renderLevel({ gameLevel }) {
     const levelEl = document.getElementById(EL_IDS.levelValue)
     levelEl.textContent = gameLevel
 }
+function renderNextLevelScore({ nextLevelScore }) {
+    const levelEl = document.getElementById(EL_IDS.nextLevelScore)
+    if (levelEl.textContent + '' !== nextLevelScore + '') {
```

Comment:
I feel like using `nextLevelScore.toString()` is more easy to understand here. AFAIK `levelEl.textContent` is guaranteed to be a `string` or `null`, so no need to cast that.

Reply:
I think ` + ''` is safer? since `.toString()` only works for certain kinds of value.
I think ` + ''` is the same as `${}`

Reply:
In this case both are equally safe. `nextLevelScore` is either `number` type or the string `'∞'`, both which have `.toString()` method on their prototypes. I'm more concerned about readability. 

i.e., I believe just writing `nextLevelScore.toString()` or `String(nextLevelScore)` is much more readable than `nextLevelScore + ''`. And, at the end of the day, readability is most important IMO

> I think `+ ''` is the same as `${}`

Pretty sure `${}` actually calls `toString` under the hood on any objects descended from `Object` (as they all have that method), and just calls the `String()` constructor if not an object (`null`, `undefined`, `42`, etc..) 

I'm less sure about `+ ''`, but I think it has the exact same behaviour as `String()` constructor